doc/network-forwards.md

```diff
@@ -1,59 +1,96 @@
 # How to configure network forwards
 
-The following network types support forwards. See each network type section for more details.
+```{note}
+Network forwards are available for the {ref}`OVN network <network-ovn>` and the {ref}`Bridge network <network-bridge>`.
+```
 
- - [bridge](#network-bridge)
- - [ovn](#network-ovn)
+Network forwards allow an external IP address (or specific ports on it) to be forwarded to an internal IP address (or specific ports on it) in the network that the forward belongs to.
 
- Network forwards allow an external IP address (or specific ports on it) to be forwarded to an internal IP address (or specific ports on it) in the network that the forward belongs to.
+This feature can be useful if you have only one public IP address, but you don't want to statically assign this address to one instance only.
+There are two different ways how you can use network forwards in this case:
+
+- Forward all traffic from the external address to the internal address of one instance.
+  This method makes it easy to move the external address to another instance by simply reconfiguring the network forward.
+- Forward traffic from different port numbers of the external address to different instances (and optionally different ports on those instances).
+  This method allows to "split up" your public IP address and expose more than one instance at a time.
```

Comment:
Above I suggested using the word "share", but we would use the same wording both here and above, so if you feel "split up" is better, then using that in place of "share" would be consistent.

Reply:
"share" is better imo - updated. :)